cle/backends/pe/pe.py

```diff
@@ -63,8 +65,13 @@ def __init__(self, *args, **kwargs):
         self._symbol_cache = self._exports # same thing
         self._handle_imports()
         self._handle_exports()
-        self.__register_relocs()
+        if self.loader.perform_relocations:
```

Comment:
I don't think this change requires making perform_relocations a public attribute. Backends should be able to access private members of the loader. the public/private distinction is only important to provide a useful interface to end-users. Or, you could think of the backends as friend classes.